.gitignore

```diff
@@ -1,15 +1,12 @@
-.coda
-.coda.json
+*yarn*
```

Comment:
Are these changes relevant to PR? It would seem tangential to update this file. Additionally, some of these wildcards seem too broad.

Reply:
I'll revert this.